lib/wanda_web/views/result_view.ex

```diff
@@ -0,0 +1,16 @@
+defmodule WandaWeb.ResultView do
+  use WandaWeb, :view
+
+  alias Wanda.Results.ExecutionResult
+
+  def render("list_results.json", %{results: results, total_count: total_count}) do
+    %{
+      items: render_many(results, WandaWeb.ResultView, "result.json", as: :result),
+      total_count: total_count
+    }
+  end
+
+  def render("result.json", %{result: %ExecutionResult{} = result}) do
+    result
```

Comment:
Here we depend on the internal domain. We could maybe at some point create mappers to provide the frontend specific example (even though it is exactly the same). Just as possibility

Reply:
well, this is not really a part of the domain since `ExecutionResult` is the persistence, since it is the ecto schema. I think the mapper you are referring to could be the view itself.

Reply:
Yes, the view is the mapper. Now we just send our persisted data, which by now would be good enough. I meant that we might want to map to a specific struct (not now).